src/theme.js

```diff
@@ -1,3 +1,122 @@
+import {black, white, gray, blue, green, orange, purple, red, yellow} from 'primer-colors'
```

Comment:
I didn't copy the raw values over, but can also add theme inline if that's helpful

Reply:
I probably don't need them, but can always add later if useful for demo